lib/modules/datasource/aws-machine-image/readme.md

```diff
@@ -1,6 +1,28 @@
 > :warning: **This datasource is _experimental_**: Be aware that its syntax and behavior may change at any time!
 
-This datasource returns the latest [Amazon Machine Image](https://docs.aws.amazon.com/en_en/AWSEC2/latest/UserGuide/AMIs.html) via the AWS API (valid credentials required).
+This datasource returns the latest [Amazon Machine Image](https://docs.aws.amazon.com/en_en/AWSEC2/latest/UserGuide/AMIs.html) via the AWS API.
+
+As this datasource relies on the AWS-SDK for JavaScript, the configuration looks similar to other AWS Tools and the common configuration options are supported, for example:
```

Comment:
```suggestion
This datasource returns the latest [Amazon Machine Image](https://docs.aws.amazon.com/en_en/AWSEC2/latest/UserGuide/AMIs.html) via the AWS API.

Because the datasource uses the AWS-SDK for JavaScript, you can configure it like other AWS Tools.
You can use common AWS configuration options, for example (partial list):
```

How about something like this?

Reply:
I like it